src/backend/distributed/Makefile

```diff
@@ -169,6 +169,8 @@ $(EXTENSION)--7.0-14.sql: $(EXTENSION)--7.0-13.sql $(EXTENSION)--7.0-13--7.0-14.
 	cat $^ > $@
 $(EXTENSION)--7.0-15.sql: $(EXTENSION)--7.0-14.sql $(EXTENSION)--7.0-14--7.0-15.sql
 	cat $^ > $@
+$(EXTENSION)--7.0-16.sql: $(EXTENSION)--7.0-15.sql $(EXTENSION)--7.0-15--7.0-16.sql
```

Comment:
Realized this should be `7.1-1`, since we're now in the 7.1 cycle. I'll fix, but waiting for any other feedback.